src/transformers/adapters/prefix_tuning.py

```diff
@@ -80,12 +80,11 @@ def __init__(
         self.dropout = nn.Dropout(self.config.dropout)
 
     def forward(self, batch_size):
-        device = next(self.parameters()).device
         key_values = (
             self.control_trans.unsqueeze(0)
             .expand(batch_size, -1)
             .view(batch_size, self.config.prefix_length, self.n_layers * 2, self.n_heads, self.n_embd_per_head)
-            .to(device)
+            .to(self.device)
```

Comment:
Please also add the mixin to the `FlatPrefixTuning` class to pass the tests.

Reply:
Hey @calpt sorry for the delay. I just added the mixin to that class.